tests/plots/test_declarative.py

```diff
@@ -395,6 +395,39 @@ def test_declarative_layers_plot_options():
     return pc.figure
 
 
+@pytest.mark.mpl_image_compare(remove_text=True, tolerance=0.009)
```

Comment:
Not sure why old 3.9 fails exactly, it doesn't show much in the result images, so I'd just bump the threshold:
```suggestion
@pytest.mark.mpl_image_compare(remove_text=True, tolerance=0.021)
```